BUILD

```diff
@@ -0,0 +1,7 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+
+pkg_tar(
+    name = "pkg_coverage",
+    srcs = glob(["coverage/*"]),
+    extension = "tar",
```

Comment:
default is `tar` already, unless yo want to use `tar.gz`

Reply:
> Should we also worry about `package_dir`?

I will put them under `/coverage` in `.tar`